docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,49 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- The `cy.log()`command will now display a line break where the `\n` character
+  is used. Addressed in
+  [#25467](https://github.com/cypress-io/cypress/pull/25467).
+- `.jsx`/`.tsx` file extensions will be utilized when generating a new empty
+  spec file inside component testing if the project contains at least one file
+  with those extensions. Addressed in
+  [#25318](https://github.com/cypress-io/cypress/pull/25318).
+- Added new `experimentalSkipDomainInjection` flag to disable Cypress from
+  setting `document.domain` on injection. Addressed in
+  [#25307](https://github.com/cypress-io/cypress/pull/25307).
+- Added new `experimentalMemoryManagement` flag to improve memory management in
+  Chromium-based browsers. Addressed in
+  [#25462](https://github.com/cypress-io/cypress/pull/25462).
+- Video output link in `cypress run` mode has been added to it's own line to
+  make the video output link more easily clickable in the terminal. Addressed in
+  [#23915](https://github.com/cypress-io/cypress/pull/23915).
+- Support `data-qa` selector in Selector Playground in addition to `data-cy`,
+  `data-test` and `data-testid`. Addressed in
+  [#25475](https://github.com/cypress-io/cypress/pull/25475).
+  - `.as()` now accepts an options argument, allowing test writers to store an
+    alias as a "query" or "static". Addressed in
+    [#25251](https://github.com/cypress-io/cypress/pull/25251).
+
+**Bugfixes:**
+
+- Fixes an issue where component tests could incorrectly treat new major
+  versions of certain dependencies as supported. Addressed in
+  [#25380](https://github.com/cypress-io/cypress/pull/25380).
+- Fixes an issue where spaces on new lines weren't kept in the test runner.
+  Addressed in [#24687](https://github.com/cypress-io/cypress/pull/24687).
+- Fixes an issue where Angular component testing projects would fail to
+  initialize if an unsupported browserslist entry was specified in the project
+  configuration. Addressed in
```

Comment:
Anytime a pull request closes an issue, it should be referenced instead of the pull request. So for this issue, it should be,
`Fixed [25312](https://github.com/cypress-io/cypress/issues/25312).

Reply:
Is this for fixes only or features and others as well?

Reply:
Sorry I just saw your comment on https://github.com/cypress-io/cypress-documentation/blob/main/docs/guides/references/changelog.mdx. Going through now and updating those!

Reply:
This should be fixed in f4a552eda